worlds/LauncherComponents.py

```diff
@@ -18,16 +18,42 @@ class Type(Enum):
 
 
 class Component:
+    """
+    A Component represents a process launchable by Archipelago Launcher, either by a User action in the GUI,
+    by resolving an archipelago://user:pass@host:port link from the Webhost, by resolving a patch file's metadata,
```

Comment:
I really think you should use the same casing in all mentions of webhost
```suggestion
    by resolving an archipelago://user:pass@host:port link from the WebHost, by resolving a patch file's metadata,
```
great idea!